Source/PeleLMeX_Plot.cpp

```diff
@@ -1100,6 +1100,64 @@ PeleLM::initLevelDataFromPlt(int a_lev, const std::string& a_dataPltFile)
   }
 }
 
+void
+PeleLM::addLevelVelocityDataFromPlt(int a_lev, const std::string& a_velPltFile)
```

Comment:
The same purpose could be accomplished by restarting from a plot file, then using the patchflowvariables user-defined function, but this is probably generally useful enough to merit just having it's own function.